lib/endpoints/class-wp-rest-comments-controller.php

```diff
@@ -899,6 +885,29 @@ public function get_item_schema() {
 				),
 			),
 		);
+
+		if ( get_option( 'show_avatars' ) ) {
```

Comment:
Okay, will fix.  In the codex it is stored as 1 and 0 integer type.  Is this changed to a boolean or should I check for 1 === get_option( 'show_avatars' )?